CONTRIBUTING.md

```diff
@@ -160,6 +160,15 @@ update /etc/hosts
 127.0.0.1 mesherylocal.layer5.io
 ```
 
+### UI Lint Rules 
+
+We are using ES-Lint to maintain code quality & consistency in our UI Code. To make sure your PR passes all the UI & ES-Lint Tests, please see below :
+
+- Remember to run `make run-ui-lint` & `make run-provider-ui-lint` if you are making changes in Meshery-UI & Provider-UI respectively.
```

Comment:
Nice. Should we take the next step and include these as subcommands called from the main `make run-ui-dev`, too?

Reply:
> Nice. Should we take the next step and include these as subcommands called from the main `make run-ui-dev`, too?

That would be great, even I was thinking the same as using a lot of make commands will be a hectic for contributor 😄

Reply:
@kushthedude are you thinking of adding that Makefile update (of command grouping) in another PR?

Reply:
Done here https://github.com/layer5io/meshery/pull/1109#pullrequestreview-424965774

Reply:
Oh, I see. I don't think I understand the connection between and the `package.json` scripts and the makefile commands. Which `package.json` scripts are executed as part of a given makefile command?

I was thinking that `make run-ui-lint` would be available as it's own command, but also implicitly called as part of an overarching command like `run-ui-dev`.

https://github.com/layer5io/meshery/blob/master/Makefile#L87

Reply:
Actually, `make run-ui-dev` runs `npm run dev` from package.json. What I have done is I have modified `npm run dev` to include a prior run to check for lint errors, as if someone who do not use `make run-ui-dev` and use `npm run dev` will also be able to see the eslint errors if there are any.

Reply:
Oh, I see. So, whether through `make` or through `npm` directly, linting will be performed. This was sitting so obviously in my face with `npm run dev;` inside of the make commands.